.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
can you maybe git ignore this?

Reply:
I was finding it pretty useful and thought others might to, since your global settings might conflict with this particular project. (E.g. my settings enforce double quotes in Typescript by default, whereas this repo does single quotes.)

Reply:
Do we have VS Code settings in any of our other repos? If not I can get rid of it, but it might be a nice convenience for developers who maintain different personal settings.

Reply:
Honestly I don't know, I don't think we have them. Usually editors will derive the settings from eslint settings rather that requiring their own settings file.  @tmeyer2115 ?

Reply:
Tom is out this week -- We don't check in personal settings like these into our repos, any rules we want to enforce are usually through eslint configurations only.